code/controlconfig/controlsconfigcommon.cpp

```diff
@@ -416,7 +416,8 @@ SCP_unordered_map<SCP_string, IoActionId> old_text = {
 	{"Custom Control 5",                        CUSTOM_CONTROL_5},
 };
 
-const char *Scan_code_text_german[] = {
+//Scan code texts are no longer needed as SDL handles the names
```

Comment:
If these char arrays are going to be eliminated, might as well delete the whole definition instead of commenting outl.  They'll still be in the git history if we ever need to find them again.